chemclipse/plugins/org.eclipse.chemclipse.chromatogram.xxd.peak.detector.supplier.firstderivative/src/org/eclipse/chemclipse/chromatogram/xxd/peak/detector/supplier/firstderivative/core/FirstDerivativePeakDetector.java

```diff
@@ -45,6 +50,23 @@
 
 @Component(service = {IMeasurementPeakDetector.class, Detector.class})
 public class FirstDerivativePeakDetector implements IMeasurementPeakDetector<FirstDerivativePeakDetectorSettings> {
+	
+	public static IMarkedIons buildFilterIons(PeakDetectorSettingsMSD peakDetectorSettings) {
```

Comment:
It would be better to place this method in PeakDetectorSettingsMSD and rename it to getFilterIons() and mark it with @JSonIgnore